### ISSUE TITLE
Set max-width on all UI elements

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4250]
### Background
Some of our UI elements don't have a max-width, which causes the page to display oddly when longer-than-expected content is introduced.

### Action requested
Add max-width to the elements listed in comments below (this is a meta-tracking ticket).

### Acceptance Criteria
UI elements have a standard render, even with long content

### Additional context
https://skylight-hq.slack.com/archives/C01F3DSC7V2/p1662740508680449

**Comment by emmastephenson** [comment]
Issue 2: Testing facility dropdown on Results page 
![image](https://user-images.githubusercontent.com/80282552/189397106-d9c9e409-8601-47a2-853b-07805940f198.png)

**Comment by vz3** [comment]
Issue 3: Test cards
(Name field should have max-width so timer doesn't drop down)

<img width="1194" alt="Screen Shot 2022-09-09 at 1 17 28 PM" src="https://user-images.githubusercontent.com/582918/189408260-0c60e0c9-ac4f-4302-83ce-c1942f1c9d43.png">

**Comment by vz3** [comment]
Issue 4: People page
<img width="1011" alt="Screen Shot 2022-09-09 at 1 21 19 PM 1" src="https://user-images.githubusercontent.com/582918/189408860-a1a9d32e-6b14-4ad3-996a-d4e3089e1b06.png">

**Comment by emmastephenson** [comment]
Also check on the strings for positive/negative on multiplex - specifically, make sure they +/- symbol isn't being broken up unexpectedly:
![image](https://user-images.githubusercontent.com/80282552/189742750-4e9a69f8-1684-4da7-ace7-efd7313dafd3.png)
(the above is MacOS/Safari)

**Comment by kenieh** [comment]
@emmastephenson Is this ticket currently in progress? If so, do we have an assigned designer pairing with an Engineer? I'm assuming this has design ramifications for some of our pages/components.

**Comment by emmastephenson** [comment]
@kenieh not currently in progress, we don't have an engineer assigned yet!

**Comment by georgehager** [comment]
@emmastephenson if this is a priority now I can pair with @kenieh next week.

**Comment by emmastephenson** [comment]
@georgehager I think that's up to @kenieh - I was hoping to get in #1427 this sprint and save this one for next time, but if this is urgent we can re-prioritize.

**Comment by kenieh** [comment]
not urgent! @georgehager we can pair when we're ready to pick this tiket up

**Comment by emmastephenson** [comment]
@georgehager to investigate if this is already being worked on by the Better Data team (#4759)

**Comment by georgehager** [comment]
@emmastephenson Nathan's work in #4759 doesn't exactly address this ticket, however it'll be useful to wait until that work is merged in. I'll put this on hold for a bit.

**Comment by georgehager** [comment]
Noting for when I pick this up:
> #2 on that list is already done because I needed to shrink that input field down to bring the results queue down to 1200px

**Comment by georgehager** [comment]
This ticket isn't strictly blocked by the #4559 but I think it'd be better to wait until that work is done as there will be some overlap.

**Comment by georgehager** [comment]
Suggested max-widths 

**Issue 1**
Manage Users sidenav - 300px (1/4 of container max width)
<img width="1224" alt="Screen Shot 2022-12-22 at 10 55 04 AM" src="https://user-images.githubusercontent.com/89797785/209172815-8e1af456-d572-4674-93a6-4506fc7562d7.png">

**Comment by georgehager** [comment]
**Issue 2** ✅  in #4559

**Comment by georgehager** [comment]
**Issue 3**

Option 1: container for name and dob has max-width 725px
<img width="1224" alt="Screen Shot 2022-12-22 at 11 05 33 AM" src="https://user-images.githubusercontent.com/89797785/209174778-5793f1ee-6387-4e5c-b02a-35a7417360b6.png">


Option 2: just name has max width

**Comment by georgehager** [comment]
**Issue #4** People Page max-width for name column is 300px

<img width="1201" alt="Screen Shot 2022-12-23 at 8 43 54 AM" src="https://user-images.githubusercontent.com/89797785/209345837-31bcbf15-7468-495d-bc78-4bb760d214ee.png">

**Comment by georgehager** [comment]
**Issue 5**

I am recommending that the bug with the `(-)` symbols getting broken in MacOS/Safari be a new ticket. After investigation, this is not a max-width issue like the other issues in this ticket. Changing the `max-width` of either the left or right column of the test card does not affect this issue. It appears to potentially be a font rendering issue in Safari. It may be possible to set a `min-width` on the labels for the radio buttons to correct the issue, however that solution is not ideal for a responsive experience. I suggest some further investigation.

cc: @emmastephenson @kenieh

**Comment by emmastephenson** [comment]
The underlying PR has been merged.
